doc/api/addons.markdown

```diff
@@ -548,7 +599,7 @@ JavaScript:
 
     #endif
 
-The implementation is similar to the above in `myobject.cc`:
+The implementation of `myobject.cc` is similar to the previous example:
```

Comment:
probably should still be "implementation in `myobject.cc`" cause it's the implementation _of_ what's outlined in `myobject.h`